### ISSUE TITLE
kobomapper does not work with JOSM 14253+

### ISSUE THREAD

**Issue by don-vip** [mossplix/kobomapper#1]
`Main` was deprecated. It's gone in [JOSM 14253](https://josm.openstreetmap.de/changeset/14253/josm/).

`Main.parent` must be replaced by `MainApplication.getMainFrame()` (introduced in [JOSM 14140](https://josm.openstreetmap.de/changeset/14140/josm)).

`Main.pref` must be almost always replaced by `Config.getPref()` (introduced in [JOSM 12847](https://josm.openstreetmap.de/changeset/12847/josm)).
Few advanced methods can only be called from `Preferences.main()` (introduced in [JOSM 14149](https://josm.openstreetmap.de/changeset/14149/josm)).

Plugin must be [updated](https://josm.openstreetmap.de/jenkins/job/JOSM-Integration/jdk=JDK8/2081/warnings3Result/type.-672070094/) and a new version released.

**Comment by don-vip** [comment]
Up. To remain a plugin proposed on the default JOSM list we require plugin maintainers to be reactive when dealing with updates.

**Comment by don-vip** [comment]
Up. Everyone updated their plugin except you. Please release a new version tomorrow.

**Comment by don-vip** [comment]
Plugin removed from default list: https://josm.openstreetmap.de/wiki/PluginsSource?action=diff&version=411

**Comment by Kiggude** [comment]
@don-vip  can I still access the Kobo mapper plugin? If yes, so how can I access it

**Comment by don-vip** [comment]
No. Authors never replied to my request. This plugin seems abandoned.

**Comment by mossplix** [comment]
Hi Vincent,

Sorry I  missed the notifications. I am going to update the plugin as needed
